### PR TITLE
Delete old tokenizer and replace them with new tokenizer

### PR DESCRIPTION
**Pull request by mpherman2**
I did a search to make sure there were not any more references to the old tokenizer,  but if you can think of any other place they were used please let me know.

### CODE REVIEW

File: src/python/bot/minimizer/utils.py [hunk 1]
```diff
@@ -130,9 +131,9 @@ def single_test_run(test_path):
   return True
 
 
-def tokenize(data, level=0):
```
Comment (mbarbella-chromium) [inline src/python/bot/minimizer/utils.py]: Is this function actually referenced anywhere? This was intended as a lightweight simple test, and I think the new tokenizer is probably overkill for what I intended here. If we can just delete it that would be ideal, but if it's still referenced in any of the main functions for minimizers it's fine to leave it as-is.

Comment (mpherman2) [reply src/python/bot/minimizer/utils.py]: https://github.com/google/clusterfuzz/blob/3722e18b8f266a5de9d4baa068fb2aab58f4501b/src/python/bot/minimizer/chunk_minimizer.py#L53

Comment (mbarbella-chromium) [reply src/python/bot/minimizer/utils.py]: Thanks for checking. Seems fine as-is, then.

**Pull request merged**